gulpfile.js

```diff
@@ -30,6 +30,9 @@ var config = {
 gulp.task('connect', function() {
 	connect.server({
 		root: ['dist'],
+		middleware: function(connect, opt) {
+			return [ historyApiFallback() ];
+		},

```

Comment:
historyApiFallback is handy because it assures index.html is loaded regardless of what's in the URL. In other words, all requests are sent to index.html so client-side routing can handle all requests.